indico/modules/events/layout/clone.py

```diff
@@ -53,3 +55,44 @@ def clone(self, new_event, options):
             db.session.add(new_image)
             db.session.flush()
             logger.info('Added image during event cloning: {}'.format(new_image))
+
+
+class LayoutCloner(EventCloner):
+    def get_options(self):
+        return {'layout': (_('Layout settings'), True, False)}
+
+    def clone(self, new_event, options):
+        if 'layout' not in options:
+            return
+        layout_settings.set_multi(new_event, layout_settings.get_all(self.event))
+        logger.info('Cloned layout settings from event {} to {}'.format(self.event, new_event))
+
+
+class MenuCloner(EventCloner):
+    def get_options(self):
+        enabled = layout_settings.get(self.event, 'use_custom_menu')
+        return {'menu': (_('Menu settings'), enabled, False)}
+
+    def clone(self, new_event, options):
+        if 'menu' not in options:
+            return
+        layout_settings.set(new_event, 'use_custom_menu', layout_settings.get(self.event, 'use_custom_menu'))
+        for menu_entry in menu_entries_for_event(self.event):
+            self._copy_menu_entry(menu_entry, new_event)
+            db.session.add(menu_entry)
+        db.session.flush()
+
+    def _copy_menu_entry(self, menu_entry, event):
+        new_menu_entry = MenuEntry(event_id=event.id)
+        for column in menu_entry.__table__.columns:
```

Comment:
Check what I did in ReminderCloner to iterate over the column attributes properly